src/main/java/core/basesyntax/figure/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.Figure;
+import core.basesyntax.enums.FigureName;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int boundMax = 20;
+    private final int boundMin = 1;
+    private final int defaultRadius = 10;
+    private final Random random = new Random();
+    private int indexFigureName;
+
+    public int randomParameters() {
```

Comment:
Methods that use only in class should be private
```suggestion
    private int randomParameters() {
```